spec/requests/api/v1/docs/tags_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+require "swagger_helper"
+
+# rubocop:disable RSpec/EmptyExampleGroup
+# rubocop:disable RSpec/VariableName
+
+RSpec.describe "Api::V1::Docs::Tags" do
+  let(:Accept) { "application/vnd.forem.api-v1+json" }
+  let(:api_secret) { create(:api_secret) }
+  let(:user) { api_secret.user }
+
+  describe "GET /tags" do
```

Comment:
The two examples below "/api/follows/tags" and "/api/tags" probably need two separate describe blocks as currently they both are under a describe block "GET /tags" which does not map to what the description for "/api/follows/tags" should read.